Draft/RFC0001-PowerShell-Based-LCM.md

```diff
@@ -63,7 +74,14 @@ foreach($resource in $resources) {
     }
 
     if ($false -eq (Invoke-DscResource @InvokeParameters -Method Test)) {
-        Invoke-DscResource @InvokeParameters -Method Set
+        if($IsWindows) {
+            Invoke-DscResource @InvokeParameters -Method Set
+        } else {
+            # run old python scripts for linux using ps job for user context
+            
```

Comment:
I would really not bother invoking Python resources there.
If there's value, let's wait for Invoke-DscResource to either do it for us or be open sourced so we can add the functionality.